src/libraries/System.Net.Requests/tests/System.Net.Requests.Tests.csproj

```diff
@@ -10,12 +10,17 @@
     <!-- the res/xml/network_security_config.xml file comes from the System.Net.TestData package -->
     <IncludeNetworkSecurityConfig Condition="'$(TargetOS)' == 'Android'">true</IncludeNetworkSecurityConfig>
   </PropertyGroup>
+  <ItemGroup>
+    <AssemblyAttribute Include="System.Runtime.Versioning.RequiresPreviewFeaturesAttribute" />
```

Comment:
This should be just `    <EnablePreviewFeatures>true</EnablePreviewFeatures>` in the PropertyGroup. It is what we do in all other tests.